File: experiment_jp_valid/mutants/if__0__DYNAMIC_PROGRAMMING_HIGH_EFFORT_VS_LOW_EFFORT_TASKS_PROBLEM.java
static int maxTasks ( int high [ ] , int low [ ] , int n ) { return Math . max ( high [ n - 1 ] + maxTasks ( high , low , ( n - 2 ) ) , low [ n - 1 ] + maxTasks ( high , low , ( n - 1 ) ) ) ; }

File: experiment_jp_valid/j2pyrawpy/if__0__DYNAMIC_PROGRAMMING_HIGH_EFFORT_VS_LOW_EFFORT_TASKS_PROBLEM.py
#!/usr/bin/env python
""" generated source for module if__0__DYNAMIC_PROGRAMMING_HIGH_EFFORT_VS_LOW_EFFORT_TASKS_PROBLEM """
class X(object):
    """ generated source for class X """
    @classmethod
    def maxTasks(cls, high, low, n):
        """ generated source for method maxTasks """
        return Math.max(high[n - 1] + cls.maxTasks(high, low, (n - 2)), low[n - 1] + cls.maxTasks(high, low, (n - 1)))

